training/creating_dataset.py

```diff
@@ -42,13 +83,149 @@ def is_path_correct(path: str) -> bool:
     """
     if not os.path.isfile(path):
         raise ValueError(f"Path {path} does not exist")
-        # return False
+
     if not (path.endswith(".db") or path.endswith(".json")):
         raise ValueError(f"Expected db or json file, got {path}")
-        # return False
+
     return True
 
 
+def split_characteristics_into_groups(
+    characteristics: list,
+) -> tuple[set[str], set[str], set[str], set[str], list[str]]:
```

Comment:
Jeżeli już robimy coś tak złożonego, to może przydałoby się zrobić dataclass na to?